packages/match/controller/src/__tests__/index.test.jsx

```diff
@@ -110,17 +123,29 @@ describe('model', () => {
       });
     });
 
+    it('returns a new object, without using the same reference', () => {
```

Comment:
object references aren't really an issue, when run in a secure environment, the `model` function will run remotely and be returns as json - so they'll never be the same reference.

Reply:
If I didn't `clone` the object the model would never come back with rows that have values. So I had to clone it, look below at my comment.